R/retry.R

```diff
@@ -16,6 +20,10 @@
 #'   \code{pause_cap} seconds.
 #' @param quiet If \code{FALSE}, will print a message displaying how long
 #'   until the next request.
+#' @param terminate_on Optional vector of numeric or integer HTTP status codes
+#'   that if found on the response will immediately stop the retries. If missing
```

Comment:
I am so sorry about the confusion, I thought you wanted to remove the entire sentence that describes what the behavior is when the parameter was NULL or missing. My mistake. 🙇 

Will replace "if missing or null" for "if null" as request right now.